benchexec/tools/depthk.py

```diff
@@ -3,12 +3,16 @@
 """
 BenchExec is a framework for reliable benchmarking.
 This file is part of BenchExec.
+
```

Comment:
Please do not add irrelevant whitespace changes in the same commit as real changes.